pkg/db/db.go

```diff
@@ -132,13 +129,13 @@ func (c *Client) isNewDB(meta metadata.Metadata) bool {
 }
 
 // Download downloads the DB file
-func (c *Client) Download(ctx context.Context, dst string) error {
+func (c *Client) Download(ctx context.Context, dst string, dbRepository string) error {
```

Comment:
What if we add WithDBRepository here?
https://github.com/aquasecurity/trivy/blob/bd243bcb1362c00dc6b64d7c311c19b11331749a/pkg/db/db.go#L26-L46

Reply:
Good idea! I've updated it 👍

Reply:
Thanks for the update.